e2e/explorer/api/src/app/etherscan/account.ts

```diff
@@ -60,17 +60,26 @@ export function txList(params: GetParams<TxList>): TransactionReceipt[] {
   return sorted.slice(offset*(page-1), offset*page);
 }
 
+/**
+ * No reward calculation on hardHat, so blockReward = 0;
+ * When a same block is mined 2 times at the same time, it create 2 parallel branch
+ * the one keep by the network is the block, the other one is the uncle.
+ * No parallel branch on hardhat so no uncles block 
+ */
 export function getMinedBlocks(params: GetParams<BlockMined>) {
   const { address, blocktype, page, offset } = params;
   if (!address) throw new Error('Error! Missing or invalid Action name');
+  if (blocktype === "uncles") throw new Error('No uncles block on local hardhat network');
 
   const minedBlocks = blocks
-    .filter(block => (block.miner === address))
-
+  .filter(block => (block.miner === address))
+  .map(minedblock => {
+    return { 
+      blockNumber: minedblock.number,
+      timeStamp: minedblock.timestamp,
+      blockReward: 0
```

Comment:
put your comment here 
```suggestion
      // No reward calculation on hardHat, so blockReward = 0;
      blockReward: 0
```